packages/playwright-test/src/cli.ts

```diff
@@ -92,7 +92,7 @@ function addTestCommand(program: Command) {
   command.description('run tests with Playwright Test');
   command.option('--browser <browser>', `Browser to use for tests, one of "all", "chromium", "firefox" or "webkit" (default: "chromium")`);
   command.option('--headed', `Run tests in headed browsers (default: headless)`);
-  command.option('--debug', `Run tests with Playwright Inspector. Shortcut for "PWDEBUG=1" environment variable and "--timeout=0 --maxFailures=1 --headed --workers=1" options`);
+  command.option('--debug', `Run tests with Playwright Inspector. Shortcut for "PWDEBUG=1" environment variable and "--timeout=0 --max-failures=1 --headed --workers=1" options`);
```

Comment:
You should also update the docs in `docs/src/test-cli-js.md`

Reply:
Thanks! Totally missed that. I updated this and grepped for `maxFailures`, which didn't yield any other results (beside the actual sourcecode).